docs/Mirrors.md

```diff
@@ -10,7 +10,7 @@ help us with this. The current list of public mirrors can be found on the
 
 You can create a public AlmaLinux mirror in a few easy steps:
 
-1. Make sure that you have enough free space: 500GB per major version is the suggested minimum.  As there are currently two supported major versions (8 and 9) the recommended minimum storage space is 1TB.  If you are also mirroring [AlmaLinux OS Kitten](https://wiki.almalinux.org/development/almalinux-os-kitten-10.html) we recommend 1.5TB.
+1. Make sure that you have enough free space: 500GB per major version is the suggested minimum.  As there are currently three supported major versions (8, 9, and 10) the recommended **minimum** storage space is 1.5TB.  2TB is more ideal.  The current steady-state of the mirror is approximately 1TB.  If you are also mirroring [AlmaLinux OS Kitten](https://wiki.almalinux.org/development/almalinux-os-kitten-10.html) we recommend 2.5TB.
```

Comment:
> As there are currently three supported major versions (8, 9, and 10)

I realize this is kind of dealt with below, but I think this should either say 4, or should indicate that we're excluding Kitten from the list.

Reply:
Kitten is not included in the stock figure because it's a separate rsync module.